src/ResultsArea/ResultsArea.js

```diff
@@ -0,0 +1,29 @@
+//imports
+import React from 'react';
+import FilterButton from '../FilterButton/FilterButton';
+import GenreArea from '../GenreArea/GenreArea';
+import '../style.css';
+
+const ResultsArea = (props) => {
+
+    const { assetsNumber, isAssetClicked, genre} = props;
+    const content = !assetsNumber && <div className='results-area' id='no-films' ></div>;
+
+    if (content) {
+        return content;
+    }
+
+    return (
+        <div className='results-area'>
+            { isAssetClicked ? <GenreArea genre={genre} /> : <p className='results-title'>{assetsNumber + ' movies found'}</p> }
```

Comment:
Please, use string template here. It would be better for performance purpose: `<p className='results-title'>{`${assetsNumber} movies found`}</p>`